File: dataset/human/c/file_2635.c
#include<stdio.h>
int main(void)
{
 char s[3];
 int i;
 int a=0,b=0,c=0;
 scanf("%s",&s[0]);
 for(i=0;i<3;i++){
     if(s[i]=='a')
        a++;
     else if(s[i]=='b')
        b++;
     else if(s[i]=='c')
        c++;
 }
 if(a==1&&b==1&&c==1)
    printf("Yes");
 else
    printf("No");
}

File: dataset/human/java/file_2635.java
import java.util.*;

public class Main {
        public static void main(String args[]) {
                Scanner sc = new Scanner(System.in);
                String s = sc.nextLine();
                String t = sc.nextLine();
                char[] s2 = s.toCharArray();
                char[] t2 = t.toCharArray();

                Arrays.sort(s2);
                Arrays.sort(t2);
                for (int i = 0;;i++) {
                        if (i == t.length()) {
                                System.out.println("No");
                                break;
                        }
                        if (i == s.length()) {
                                System.out.println("Yes");
                                break;
                        }
                        if (s2[i] < t2[t.length() - i - 1]) {
                                System.out.println("Yes");
                                break;
                        } else if (s2[i] > t2[t.length() - i - 1]) {
                                System.out.println("No");
                                break;
                        }
                }
        }
}
